react-native-libraries.json

```
@@ -4224,6 +4224,19 @@
     "expoGo": true,
     "examples": ["https://github.com/jibraniqbal666/react-native-fb-image-grid/tree/master/example"]
   },
+  {
+    "githubUrl": "https://github.com/developit/mitt",
+    "examples": [
+     "http://codepen.io/developit/pen/rjMEwW?editors=0110"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "hasNativeModule": false,
+    "newArchitecture": false,
+    "npmPkg": "mitt"
```

Hey @jgallaway2011, thanks for adding new library to the directory! 👍 

Few things, before we can merge the entry in:
* let's remove invalid and redundant fields
  ```suggestion
      "expoGo": true
  ```
* let's move the entry to the end of file (we use the order for "Recently Added" sort order

For more details check out:
* https://github.com/react-native-community/directory?tab=readme-ov-file#how-do-i-add-a-library